src/sentry/tasks/merge.py

```diff
@@ -257,11 +257,16 @@ def merge_objects(models, group, new_group, limit=1000,
                             )
                     elif model == GroupTagValue:
                         with transaction.atomic(using=router.db_for_write(model)):
```

Comment:
This is unrelated to this specific change, but I'm not really sure why this block was encapsulated in a transaction context manager, since it was only executing a single query…?